### PR TITLE
Add: support common labels for all resources

### PR DESCRIPTION
**Pull request by thib-mary**
Usefull when we have a policies manager as Kyverno.

Signed-off-by: Flaagada <mary.thibault2@gmail.com>

#### Overview

Support a common labels for all resources.

#### What this PR does / why we need it

When we used a policies manager as Kyverno, it's usefull to be able to add a common labels on all resources created by the chart in order to respect the policy.

#### Checklist

- [x] Change log updated in `Chart.yaml` (see the contributing guide for details)
- [x] Chart version bumped in `Chart.yaml` (see the contributing guide for details)
- [x] Documentation regenerated by running `make docs`

### CODE REVIEW

**Comment by electrosenpai** [comment]
Hello @sagikazarmark can we have by chance, a feedback on this pull request please ?  🙏

File: charts/dex/values.yaml [hunk 1]
```diff
@@ -5,6 +5,10 @@
 # -- Number of replicas (pods) to launch.
 replicaCount: 1
 
+# -- Labels to apply to all resources
```
Comment (sagikazarmark) [inline charts/dex/values.yaml:8]:
```suggestion
# -- Labels to apply to all resources and selectors.
```

File: charts/dex/README.md [hunk 2]
```diff
@@ -111,6 +111,7 @@ ingress:
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | replicaCount | int | `1` | Number of replicas (pods) to launch. |
+| commonLabels | object | `{}` | Labels to apply to all resources |
```
Comment (sagikazarmark) [inline charts/dex/README.md:114]: Please regenerate the docs as described in the contribution guide.

**Comment by sagikazarmark** [comment]
Please follow the contribution guide to fix the failing build. Thanks!

**Comment by yann-soubeyrand** [comment]
@thib-mary I’m not sure to understand why I’m pinged here :thinking: Can you give me some context please?

**Comment by yann-soubeyrand** [comment]
OK, no problem! I had a look on your PR though, and modifying the selector labels is a breaking change I think. The selector field of a deployment is immutable IIRC. I think you should modify `dex.labels` instead.

**Review by sagikazarmark — APPROVED** [review]

**Pull request merged**